apps/emqx_psk/src/emqx_psk_schema.erl

```diff
@@ -22,12 +22,15 @@
 -include_lib("hocon/include/hoconsc.hrl").
 
 -export([
+    namespace/0,
     roots/0,
     fields/1
 ]).
 
 -import(emqx_schema, [sc/2]).
 
+namespace() -> psk.
```

Comment:
```suggestion
namespace() -> authentication
```

Reply:
Or authn

Reply:
authn-psk is better 
I saw `authn-jwt` authn-mysql` etc.